### ISSUE TITLE
time partition not working for firebase extension(0.1.49)

### ISSUE THREAD

**Issue by ishu-pmy** [firebase/extensions#2091]
I have tried to install the extension **Stream Firestore to BigQuery** (0.1.49) with time partitioned. It is showing processing state as per screenshot. Furter is does not create any table in BigQuery. 
For the old version(0.1.48), time partitioned is working fine.
But it is very critical here to know the reason why extension with time partition is not working because we have more then 50+ extensions which are installed with time partitioned tables and a lot of our data flow depends on this.

![firebase extension](https://github.com/firebase/extensions/assets/169628655/e6c7f940-6cda-4749-a963-b54bbd8d7c12)

**Comment by ishu-pmy** [comment]
Hi @dackers86 ,
Here is the 'example.png' showing the configuration.
![example config](https://github.com/firebase/extensions/assets/169628655/592e456a-b0b5-413a-b439-a0f562ee2cac)

**Comment by pr-Mais** [comment]
The reason is "BigQuery Time Partitioning column name" is using a reserved name `timestamp`, this causes the creation of the table to fail, try using any name other than `timestamp`.

**Comment by ishu-pmy** [comment]
@pr-Mais 
I have few concerns here:
1) Is 'timestamp' only reserved for the latest version(0.1.49) only? Because I have created many extensions in which I have used this    reserved name (timestamp) and is working fine.
2) If this is the case then how can I update the older versions to latest one as all extensions I am using have 'timestamp' name for the time partitioning.
3) When I create the extension(0.1.49 - with partitioned option) there is always suggestion to select 'timestamp'. If it is reserved then why there is option to select 'timestamp' as partitioning parameter.
